operations/enterprises.md

```diff
@@ -631,3 +631,67 @@ Adds a new company to the enterprise.
 ### Response
 
 Same structure as in [Get all companies](enterprises.md#get-all-companies) operation.
+
+## Update company
+
+Updates information about a company. Note that if any of the fields are sent as `null`, it won't clear the field in Mews. If the `Value` within the object is sent as `null`, the field will be cleared in Mews.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/companies/update`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "CompanyId": "9ebf6446-64c3-4f85-943c-aa540092b963",
+    "Name": {
+        "Value": "Golden Tech Company"
+    },
+    "MotherCompanyId": null,
+    "Identifier": null,
+    "TaxIdentifier": null,
+    "AdditionalTaxIdentifier": null,
+    "BillingCode": { 
+        Value: null
+    }
+    "AccountingCode": {
+        "Value": "1234"
+    }
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `CompanyId` | string | required | Unique identifier of the [Company](enterprises.md#company). |
+| `Name` | [String UpdateValue](enterprises.md#update-value) | optional | Name of the company \(or `null` if mother company should not be updated). |
+| `MotherCompanyId` | [String Update Value](enterprises.md#update-string-value) | optional | Unique identifier of the mother company \(or `null` if mother company should not be updated). |
+| `Identifier` | [String Update Value](enterprises.md#update-string-value) | optional | Identifier of the company (e.g. legal identifier) \(or `null` if mother company should not be updated). |
+| `TaxIdentifier` | [String Update Value](enterprises.md#update-string-value) | optional | Tax identification number of the company \(or `null` if mother company should not be updated). |
+| `AdditionalTaxIdentifier` | [String Update Value](enterprises.md#update-string-value) | optional | Additional tax identifer of the company \(or `null` if mother company should not be updated). |
+| `BillingCode` | [String Update Value](enterprises.md#update-string-value) | optional | Billing code of the company \(or `null` if mother company should not be updated). |
+| `AccountingCode` | [String Update Value](enterprises.md#update-string-value) | optional | Accounting code of the company \(or `null` if mother company should not be updated). |
+
+#### Update string value
```

Comment:
"String value update" - entities should be nouns, "Update string value" is rather command or verb or something like that...

Reply:
this commend is not resolved.
and bellows names should be renamed also.